ocaml/src/tools/etcd.ml

```diff
@@ -0,0 +1,48 @@
+(*
+Copyright 2015 iNuron NV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*)
+open Lwt.Infix
+
+let parse_url url =
+  let host,port,path =
+    Scanf.sscanf url "%s@:%i/%s" (fun h i p -> h,i,p)
```

Comment:
This means we only handle uri's with 1 etcd peer? Might be a problem down further down the road?